lib/src/mock.dart

```diff
@@ -528,15 +526,15 @@ class VerificationResult {
 
 typedef dynamic Answering(Invocation realInvocation);
 
-typedef VerificationResult Verification(matchingInvocations);
+typedef VerificationResult _Verification(matchingInvocations);
```

Comment:
I think this has to stay public, yes?

Verification verification = verify(foo.bar(captureAny));
var args = verification.captured;
verification.called(1);